proxy/contracts/mainnet/MessageProxyForMainnet.sol

```diff
@@ -100,6 +100,20 @@ contract MessageProxyForMainnet is SkaleManagerClient, MessageProxy, IMessagePro
         uint256 newValue
     );
 
+    /**
+     * @dev Emitted when the schain is paused
+     */
+    event SchainPaused(
+        bytes32 schainHash
```

Comment:
Let's add `indexed` to this parameter and for a next event too. To make search more efficient.

Reply:
Added.